src/umessage/umessagebuilder.rs

```diff
@@ -425,13 +410,12 @@ impl UMessageBuilder {
     ///
     /// ```rust
     /// use up_rust::{UMessageBuilder, UMessageType, UPayloadFormat, UPriority, UUIDBuilder, UUri};
-
     ///
     /// # fn main() -> Result<(), Box<dyn std::error::Error>> {
+    /// # let request_msg_id = UUIDBuilder::build();
```

Comment:
should not be fenced out FMPOV

Reply:
Same comment as [here](https://github.com/eclipse-uprotocol/up-rust/pull/90#discussion_r1571197600).